app/models/registerable_edition.rb

```diff
@@ -89,8 +89,6 @@ def published?
   end
 
   def detailed_guide_paths
-    old_slug = slug.sub(%r{^guidance/}, '')
-    ["/#{old_slug}"] + edition.non_english_translations.map { |t| "/#{edition.slug}.#{t.locale}" } +
-      ["/#{slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
+    ["/#{slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
```

Comment:
This is slightly odd to read as it uses two different slug methods which produce different kinds of slug. Would it be better as either 

```ruby
["/#{slug}"] + edition.non_english_translations.map { |t| "/#{slug}.#{t.locale}" }
```

or 

```ruby
["/guidance/#{edition.slug}"] + edition.non_english_translations.map { |t| "/guidance/#{edition.slug}.#{t.locale}" }
```

Which I think (and could be wrong) is the intent behind the line.